elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -417,6 +440,7 @@ protected ElideResponse handleRequest(boolean isReadOnly, User user,
             log.error("Error or exception uncaught by Elide", e);
             throw e;
         } finally {
+            elideCompleteQuery(requestScope, response);
```

Comment:
So the response is often set inside the exception handling logic.  For all exceptions, the response here will be null.

Reply:
Fixed in most recent commit